From f473b9244524ae13dc328875cbbf178e78d1105b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Timoth=C3=A9e=20Ravier?= <tim@siosm.fr>
Date: Fri, 25 Apr 2025 11:49:37 +0200
Subject: [PATCH 1/2] appdata: Add developer, oars rating, launchable, releases

Needed to be able to publish on Flathub but should also make the
application more attractive in applications stores such as GNOME
Software and Plasma Discover.

See: https://github.com/tibirna/qgit/issues/149
See: https://github.com/flathub/flathub/pull/6389
---
 qgit.appdata.xml | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/qgit.appdata.xml b/qgit.appdata.xml
index 3b97cd9..c312026 100644
--- a/qgit.appdata.xml
+++ b/qgit.appdata.xml
@@ -14,12 +14,21 @@
       <li>Quick access to git operations</li>
     </ul>
   </description>
+  <developer id="org.tibirna">
+    <name>Cristian Tibirna</name>
+  </developer>
   <screenshots>
     <screenshot type="default"/>
   </screenshots>
   <url type="homepage">https://github.com/tibirna/qgit</url>
   <url type="bugtracker">https://github.com/tibirna/qgit/issues</url>
+  <launchable type="desktop-id">qgit.desktop</launchable>
   <provides>
     <binary>qgit</binary>
   </provides>
+  <releases>
+    <release version="2.10" date="2022-01-09"/>
+    <release version="2.9" date="2019-08-24"/>
+  </releases>
+  <content_rating type="oars-1.1" />
 </component>
-- 
2.49.0

